projects/instanceimagenav/README.md

```diff
@@ -0,0 +1,125 @@
+# Instance ImageNav on Stretch
+
+This directory contains scripts to demo our modular InstanceImageNav model in Habitat and on a Stretch robot.
+
+In short, our method explores the environment using **frontier-based exploration (FBE)**. A SuperGLue-based **keypoint matching** system is used to detect if the goal instance is visible. Upon positive detection, keypoint correspondences localize the goal instance with the aid of **Detic instance segmentation**. Finally, an **analytic local navigator** conveys the agent to the goal.
+
+## Environment Setup
+
+These are the installation directions that work for me.
+
+Components:
+
+- Habitat v0.2.3 (Sim & Lab)
+- Detic
+- SuperGlue
+- HM3D Scene Dataset
+- 2023 Habitat Instance ImageNav Challenge Dataset
+
+```bash
+git clone -b imagenav --recurse-submodules git@github.com:jacobkrantz/home-robot.git
```

Comment:
Imo we should also remind users to `git submodule update --init --recursive` if they have `home_robot` installed already.